### DISCUSSION TITLE
Fluent System design kit for MAUI

### DISCUSSION THREAD

**Discussion by DmitryBorodiy** [dotnet/maui#25807] [Ideas]
### Description

It would be very cool to see the official Fluent Design app development kit on MAUI. On Microsoft's official Fluent Design website, you can see that they provide a set for each platform, but on MAUI I couldn't find any pre-made stylizations or controls for Fluent Design. Material design is there, but Fluent isn't?

https://www.figma.com/community/file/836835062056249539/microsoft-fluent-android

### Public API Changes

The Fluent Design kit must be on MAUI. 

### Intended Use-Case

In my app I want to see Fluent Design on all platforms.

**Comment by mattleibow** [comment]
This may be something for the community toolkit to create a style set. 

Also, this is a site with controls: https://fluent2.microsoft.design/
